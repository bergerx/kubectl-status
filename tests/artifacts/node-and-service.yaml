apiVersion: v1
items:
- apiVersion: v1
  kind: Node
  metadata:
    annotations:
      kubeadm.alpha.kubernetes.io/cri-socket: unix:///var/run/cri-dockerd.sock
      node.alpha.kubernetes.io/ttl: "0"
      volumes.kubernetes.io/controller-managed-attach-detach: "true"
    creationTimestamp: "2024-08-06T08:36:21Z"
    labels:
      beta.kubernetes.io/arch: amd64
      beta.kubernetes.io/os: linux
      kubernetes.io/arch: amd64
      kubernetes.io/hostname: minikube
      kubernetes.io/os: linux
      minikube.k8s.io/commit: 5883c09216182566a63dff4c326a6fc9ed2982ff-dirty
      minikube.k8s.io/name: minikube
      minikube.k8s.io/primary: "true"
      minikube.k8s.io/updated_at: 2024_08_06T09_36_23_0700
      minikube.k8s.io/version: v1.33.1
      node-role.kubernetes.io/control-plane: ""
      node.kubernetes.io/exclude-from-external-load-balancers: ""
    name: minikube
    resourceVersion: "8978"
    uid: 8319502f-5189-46fc-a54f-184f79a1cff5
  spec:
    podCIDR: 10.244.0.0/24
    podCIDRs:
    - 10.244.0.0/24
  status:
    addresses:
    - address: 192.168.49.2
      type: InternalIP
    - address: minikube
      type: Hostname
    allocatable:
      cpu: "16"
      ephemeral-storage: 973983276Ki
      hugepages-1Gi: "0"
      hugepages-2Mi: "0"
      memory: 32587744Ki
      pods: "110"
    capacity:
      cpu: "16"
      ephemeral-storage: 973983276Ki
      hugepages-1Gi: "0"
      hugepages-2Mi: "0"
      memory: 32587744Ki
      pods: "110"
    conditions:
    - lastHeartbeatTime: "2024-08-07T07:45:09Z"
      lastTransitionTime: "2024-08-06T08:36:20Z"
      message: kubelet has sufficient memory available
      reason: KubeletHasSufficientMemory
      status: "False"
      type: MemoryPressure
    - lastHeartbeatTime: "2024-08-07T07:45:09Z"
      lastTransitionTime: "2024-08-06T08:36:20Z"
      message: kubelet has no disk pressure
      reason: KubeletHasNoDiskPressure
      status: "False"
      type: DiskPressure
    - lastHeartbeatTime: "2024-08-07T07:45:09Z"
      lastTransitionTime: "2024-08-06T08:36:20Z"
      message: kubelet has sufficient PID available
      reason: KubeletHasSufficientPID
      status: "False"
      type: PIDPressure
    - lastHeartbeatTime: "2024-08-07T07:45:09Z"
      lastTransitionTime: "2024-08-06T08:36:21Z"
      message: kubelet is posting ready status
      reason: KubeletReady
      status: "True"
      type: Ready
    daemonEndpoints:
      kubeletEndpoint:
        Port: 10250
    images:
    - names:
      - registry.k8s.io/etcd@sha256:44a8e24dcbba3470ee1fee21d5e88d128c936e9b55d4bc51fbef8086f8ed123b
      - registry.k8s.io/etcd:3.5.12-0
      sizeBytes: 149347661
    - names:
      - registry.k8s.io/kube-apiserver@sha256:6b8e197b2d39c321189a475ac755a77896e34b56729425590fbc99f3a96468a3
      - registry.k8s.io/kube-apiserver:v1.30.0
      sizeBytes: 116552324
    - names:
      - registry.k8s.io/kube-controller-manager@sha256:5f52f00f17d5784b5ca004dffca59710fa1a9eec8d54cebdf9433a1d134150fe
      - registry.k8s.io/kube-controller-manager:v1.30.0
      sizeBytes: 111113187
    - names:
      - registry.k8s.io/kube-proxy@sha256:ec532ff47eaf39822387e51ec73f1f2502eb74658c6303319db88d2c380d0210
      - registry.k8s.io/kube-proxy:v1.30.0
      sizeBytes: 84675401
    - names:
      - registry.k8s.io/kube-scheduler@sha256:2353c3a1803229970fcb571cffc9b2f120372350e01c7381b4b650c4a02b9d67
      - registry.k8s.io/kube-scheduler:v1.30.0
      sizeBytes: 61969366
    - names:
      - registry.k8s.io/coredns/coredns@sha256:1eeb4c7316bacb1d4c8ead65571cd92dd21e27359f0d4917f1a5822a73b75db1
      - registry.k8s.io/coredns/coredns:v1.11.1
      sizeBytes: 59820619
    - names:
      - gcr.io/k8s-minikube/storage-provisioner@sha256:18eb69d1418e854ad5a19e399310e52808a8321e4c441c1dddad8977a0d7a944
      - gcr.io/k8s-minikube/storage-provisioner:v5
      sizeBytes: 31465472
    - names:
      - registry.k8s.io/pause@sha256:7031c1b283388d2c2e09b57badb803c05ebed362dc88d84b480cc47f72a21097
      - registry.k8s.io/pause:3.9
      sizeBytes: 743952
    nodeInfo:
      architecture: amd64
      bootID: 347b6358-8163-4429-b24d-c5b7841e4b41
      containerRuntimeVersion: docker://26.1.1
      kernelVersion: 6.6.41-1-MANJARO
      kubeProxyVersion: v1.30.0
      kubeletVersion: v1.30.0
      machineID: d6f70d223fef420cbd26ccac4bd0412b
      operatingSystem: linux
      osImage: Ubuntu 22.04.4 LTS
      systemUUID: d91167d3-7a3a-499e-aaaa-6c952fc10a74
- apiVersion: v1
  kind: Service
  metadata:
    creationTimestamp: "2024-08-06T08:36:22Z"
    labels:
      component: apiserver
      provider: kubernetes
    name: kubernetes
    namespace: default
    resourceVersion: "198"
    uid: ef2ff9cf-09e9-4bd6-ba01-3daf3e8886c6
  spec:
    clusterIP: 10.96.0.1
    clusterIPs:
    - 10.96.0.1
    internalTrafficPolicy: Cluster
    ipFamilies:
    - IPv4
    ipFamilyPolicy: SingleStack
    ports:
    - name: https
      port: 443
      protocol: TCP
      targetPort: 8443
    sessionAffinity: None
    type: ClusterIP
  status:
    loadBalancer: {}
kind: List
metadata:
  resourceVersion: ""
