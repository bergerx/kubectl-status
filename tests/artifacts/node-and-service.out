
Node/minikube, created 1m ago
  linux Ubuntu 22.04.4 LTS (amd64), kernel 6.6.41-1-MANJARO, kubelet v1.30.0, kube-proxy v1.30.0
  images 8
  Current: Resource is Ready
  MemoryPressure KubeletHasSufficientMemory, kubelet has sufficient memory available for 1m
  DiskPressure KubeletHasNoDiskPressure, kubelet has no disk pressure for 1m
  PIDPressure KubeletHasSufficientPID, kubelet has sufficient PID available for 1m
  Ready KubeletReady, kubelet is posting ready status for 1m
  addresses: InternalIP=192.168.49.2 Hostname=minikube
  allocatable: pods:110, cpu:16, mem:33.3GB, ephemeral-storage:997.3GB
  capacity: pods:110, cpu:16, mem:33.3GB, ephemeral-storage:997.3GB

Service/kubernetes -n default, created 1m ago
  Current: Service is ready
  Missing Endpoint: Service has no matching endpoint.
